xml/CRSChannelCode.xml

```diff
@@ -1413,8 +1434,8 @@
         <codelist-item status="active" activation-date="2015-12-07">
             <code>46008</code>
             <name>
-                <narrative>Andean Development Corporation</narrative>
-                <narrative xml:lang="fr">Société andine de développement</narrative>
+                <narrative>Development Bank of Latin America</narrative>
```

Comment:
This looks correct: https://en.wikipedia.org/wiki/CAF_%E2%80%93_Development_Bank_of_Latin_America